Detectors/ZDC/simulation/src/Detector.cxx

```diff
@@ -2398,6 +2430,22 @@ void Detector::BeginPrimary()
   resetHitIndices();
 
   mCurrentPrincipalParticle = *stack->GetCurrentTrack();
+
+#ifdef ZDC_FASTSIM_ONNX
+  if (o2::zdc::ZDCSimParam::Instance().useZDCFastSim && mFastSimModel != nullptr) {
```

Comment:
and where do you invoke the model ? Is this not done yet?

Reply:
It is done at line 2446.

Reply:
mFastSimResults.push_back(mFastSimModel->getChannels(particle));

Reply:
Ok I see. Thanks.